test/actor/DemoActorTTLStateImpl.ts

```diff
@@ -0,0 +1,24 @@
+import { AbstractActor } from "../../src";
```

Comment:
Please add the copyright header (you can take it from actors.test.ts) to new files.